test/integration/dialects/postgres/query-interface.test.js

```diff
@@ -18,6 +18,68 @@ if (dialect.match(/^postgres/)) {
       this.queryInterface = this.sequelize.getQueryInterface();
     });
 
+    describe('createSchema', function () {
+      beforeEach( function () {
+        //make sure we don't have a pre-existing schema called testSchema.
+        return this.queryInterface.dropSchema('testschema')
+        .catch( err => {});
```

Comment:
catch can be on the same line but you can also simply do `return this.queryInterface.dropSchema('testschema').reflect()` when ignoring errors